src/components/rocketTable/RocketTable.test.tsx

```diff
@@ -27,7 +27,7 @@ describe("RocketTable component", () => {
       const table = await findByTestId("rockets");
       const { getAllByRole } = within(table);
       expect(getAllByRole("columnheader")).toHaveLength(3);
-      expect(getAllByRole("row")).toHaveLength(5);
+      expect(getAllByRole("row")).toHaveLength(31);
```

Comment:
This is still making a real API call. (As you mentioned above, this makes your test slow and fragile.) Please update this test so that the API call is mocked, just like you did in `src/api/rockets/index.test.ts`.

I might add two more "integration-y" tests here. "With no results", where your axios mock returns an empty array of rockets, and "With errors", where your axios mock returns an error. You could then get rid of the two super fake tests below.

There's an example of mocking a network error [here](https://www.bha.ee/how-to-mock-axios-in-jest-with-typescript/).